docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG PHP_VERSION=7.2
+FROM php:${PHP_VERSION}-alpine
+
+ARG XDEBUG=0
+
+RUN apk add --no-cache \
+	autoconf \
+	make \
+	g++ \
+	bash \
+	git \
+	openssl-dev
+
+RUN echo -e 'memory_limit=2G' > /usr/local/etc/php/conf.d/memory.ini
+RUN set -o pipefail && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+RUN if [ ${PHP_VERSION:0:3} == "5.6" ] ; then \
```

Comment:
I was trying it before, and I think we can just have here `$PHP_VERSION == "5.6"`

Reply:
No-can-do.  Removing the substr and brackets breaks the build.

Reply:
Docker installs v 5.6.16 or something so it's not about the parameter Docker accepts, it's about the value Travis supplies.